pkg/cmd/server/api/v1/types.go

```diff
@@ -71,15 +71,21 @@ type NodeConfig struct {
 	IPTablesSyncPeriod string `json:"iptablesSyncPeriod"`
 
 	// VolumeConfig contains options for configuring volumes on the node.
-	VolumeConfig VolumeConfig `json:"volumeConfig"`
+	VolumeConfig NodeVolumeConfig `json:"volumeConfig"`
 }
 
-// VolumeConfig contains options for configuring volumes on the node.
-type VolumeConfig struct {
+// NodeVolumeConfig contains options for configuring volumes on the node.
+type NodeVolumeConfig struct {
 	// LocalQuota contains options for controlling local volume quota on the node.
 	LocalQuota LocalQuota `json:"localQuota"`
 }
 
+// MasterVolumeConfig contains options for configuring volume plugins in the master node.
+type MasterVolumeConfig struct {
+	// DisableDynamicProvisioning is a boolean that toggles dynamic provisioning off when true
+	DisableDynamicProvisioning bool `json:"disableDynamicProvisioning"`
```

Comment:
run `hack/test-go.sh pkg/cmd/server/api/v1` and update the expected config in types_test.go